dom0/sd-dom0-files.sls

```diff
@@ -52,6 +52,25 @@ dom0-workstation-rpm-repo:
     - require:
       - file: dom0-rpm-test-key
 
+dom0-workstation-templates-repo:
+  # Using file.blockreplace because /etc/qubes/repo-templates/ is not a .d
+  # style directory, and qvm.template_installed:fromrepo seems to only support
+  # using a repo from this file. Installing manually via a cli-command-instead?
```

Comment:
I think you're right that `qvm.template_installed` does not yet support an alternate repo file location. Another possibility if we want to avoid touching the qubes .repo files directly is using 
```
dom0-install-securedrop-workstation-template:
  cmd.run:
    - name: qvm-template install securedrop-workstation-buster --repo-files /etc/qubes/repo-templates/securedrop-workstation-template.repo
    - require:
      - file: securedrop-workstation-template-key  # either in /etc/pki or in /etc/qubes/repo-templates/keys
      - file: securedrop-workstation-template-repo # separate template .repo file
```